sdk/provisioning/Azure.Provisioning/src/Construct.cs

```diff
@@ -124,6 +121,17 @@ public IEnumerable<Resource> GetResources(bool recursive = true)
             return result;
         }
 
+        /// <inheritdoc/>
+        public IEnumerable<Resource> GetExistingResources(bool recursive = true)
```

Comment:
what does this do vs GetResources?

Reply:
This gets only the resources added via UseExisting

Reply:
If a resource knows that it is existing, then I would leave this off for now and let the consumer filter themselves until we have a real need.